cf/irsa-api-role.yaml

```diff
@@ -97,6 +97,7 @@ Resources:
                 Resource:
                   - !Sub "arn:aws:s3:::cell-sets-${Environment}-${AWS::AccountId}/*"
                   - !Sub "arn:aws:s3:::biomage-originals-${Environment}-${AWS::AccountId}/*"
+                  - !Sub "arn:aws:s3:::cellenics-cell-level-metadata-${Environment}-${AWS::AccountId}/*"
```

Comment:
why adding `cellenics` in the name of the bucket when no other buckets have `cellenics` in their name?

Reply:
i dont know, the bucket was added in a previous ticket.